package.xml

```diff
@@ -26,4 +26,6 @@
   <run_depend>nodelet</run_depend>
   <run_depend>tf</run_depend>
 
+  <test_depend>roslaunch</test_depend>
```

Comment:
Ok, but I suggest this time that we don't define any dependency to roslaunch here.
Usually I prefer to not relying on transitive dependency for stability. On the other hand, fewer dependency is ideal for less maintenance effort. In this case the dependency-chain to `roslaunch` seems essential enough. Curious to see test result without this.

```
$ rospack depends-why --target=roslaunch rgbd_launch
Dependency chains from rgbd_launch to roslaunch:
* rgbd_launch -> depth_image_proc -> tf2_ros -> actionlib -> rostest -> roslaunch 
* rgbd_launch -> tf -> roswtf -> roslaunch 
* rgbd_launch -> tf -> tf2_ros -> actionlib -> rostest -> roslaunch
```